eks-cijenkinsio-agents-2.tf

```diff
@@ -48,6 +48,13 @@ module "cijenkinsio_agents_2" {
         }
       }
     }
+
+    # see https://docs.aws.amazon.com/eks/latest/userguide/creating-access-entries.html
```

Comment:
```suggestion
    },
```

Reply:
- The comma `,` is not mandatory in Terraform for [maps](https://developer.hashicorp.com/terraform/language/expressions/types#maps-objects) but adding it to stay homogeneous with other maps in this project
- Unneeded empty line (since its a map, we should keep it grouped)
- Unneeded comment (it does not provide any value to the reader: if reader does not know what is an access entry, then they should read the doc)